### PR TITLE
Welcome screen buttons modified and background added. Closes #2

### LINKED ISSUE #2: Modify Welcome Screen
Welcome screen consists of 2 buttons
## Login & Register
Add Css to the buttons, Add Touchable Opacity
## Unify button size and style

### PR DESCRIPTION
**Pull request by snowmanstark**
Close #2

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "react": "16.8.3",
     "react-dom": "16.8.3",
     "react-native": "https://github.com/expo/react-native/archive/sdk-35.0.0.tar.gz",
+    "react-native-button": "^2.4.0",
```
Comment (dcipher00) [inline package.json:16]: @snowmanstark thanks for the contribution, but do we really need a new package "react-native-button" when we could have used `<TouchableOpacity />` enclosing `<Button />` or `<Text />`

File: app.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
     ],
     "version": "1.0.0",
     "orientation": "portrait",
-    "icon": "./assets/icon.png",
```
Comment (dcipher00) [inline app.json]: @snowmanstark i dont really understand why we need to do this.

Comment (snowmanstark) [reply app.json]: Due to errors saying the icon size is 192x110.

Comment (dcipher00) [reply app.json]: you should try `expo optimize` it should resolve your issue.

File: package.json (hunk 1 above)
Comment (snowmanstark) [reply package.json:16]: When using `<TouchableOpacity />` the button had to be in a container, and using `<TouchableOpacity />` we would have to tinker around setting the buttons in every page wherever we use buttons.

Comment (dcipher00) [reply package.json:16]: okay looks good thanks.

File: app.json (hunk 2 above)
Comment (snowmanstark) [reply app.json]: Even after `expo optimize` the console shows the error mesage 
`Error: Problems validating asset fields in app.json. See https://docs.expo.io/
 • Field: icon - image should be square, but the file at './assets/icon.png' has dimensions 192x110.`

**Pull request merged**